Sources/RichTextKit/Extensions/String+Paragraph.swift

```diff
@@ -59,7 +59,7 @@ public extension String {
      after the newline char, not the newline itself.
      */
 
-    func findIndexOfNextParagraphOrEndofCurrent(from location: UInt) -> UInt {
+    func findIndexOfNextParagraphOrEndOfCurrent(from location: UInt) -> UInt {
```

Comment:
hmm ho come this didn't merge with previous PR?

Reply:
Uhm not sure 🤔